src/marketing/recommendation-change-source.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Fetch Product Recommendations from a Different Source
+tag: product-recommendations
+ee_only: true
+---
+
+When you [configure a SaaS environment]({% link configuration/services/saas.md %}) for Recommendations, that SaaS environment collects catalog data and storefront behavioral data. [Adobe Sensei](https://www.adobe.com/sensei.html) analyzes that data and computes product associations used to serve Product Recommendations.
+
+On non-production environments like testing or staging, there is often not much storefront behavioral data (unless automated) to serve Recommendations. Actual shopper behavior at scale is captured only on the production environment.
+
+Magento allows you to use the storefront behavioral data from your production environment on other, non-production environments. Using this data inside non-production environments allows you to preview which recommendations your shoppers will see and experiment with different recommendation types and placement locations.
```

Comment:
"Magento allows you to use product recommendations computed in your product environment on other, non-prod.... "
(we don't use behavioral data directly)